### ISSUE TITLE
Links have extraneous content (JAWS only).

### ISSUE THREAD

**Issue by mbstacy** [culibraries/ir-scholar#59]
When using JAWS and navigating through the limiting options with the up and down arrow keys, JAWS reads each option as a heading, but then it appears that the “greater than” icon is also being read as a heading, since JAWS reads, “Creator heading level three” [down arrow to the greater than icon] and JAWS reads “heading level 3” again.
 
Since the entire element is a button, the “greater than” icon should be hidden from the screen reader. NVDA did not have the same issue, all links were read as a heading and there was not an extra heading level voiced.
